src/rust/catloop/futures/connect.rs

```diff
@@ -88,7 +88,6 @@ async fn send_connection_request(
     let handle: TaskHandle = {
         // Get scheduler handle from the task id.
         catmem.borrow().from_task_id(qt)?
-        // Drop the mutable reference because we might yield.
```

Comment:
I left this here in case someone is confused about why I put that function call inside a scope.

Reply:
Merging this now without the comment. There is another instance in the same file which doesn't have the comment. If we decide to get it back in the future, we should do it at all the places.